gocd/templates/bash/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+eval $(/devinfra/scripts/regions/project_env_vars.py --region="${SENTRY_REGION}")
+/devinfra/scripts/k8s/k8stunnel
+/devinfra/scripts/k8s/k8s-deploy.py \
+  --type="statefulset" \
```

Comment:
over at https://github.com/getsentry/ops/pull/10699/files#diff-ea11561f4a8b8518b53a0b6792e97171c75055f65fe95ee3dbf2b9873e485299R24,
this is defined as a regular `deployment` and not a `statefulset` like symbolicator.